### PR TITLE
Quick Little Poster

### PR DESCRIPTION
**Pull request by Fungal-Infection**
## About the PR 

Adds a funky little poster about a space Opera
image

## Why's this needed? 

The funny jestr hat !!!

### CODE REVIEW

File: code/obj/decal/poster.dm [hunk 1]
```diff
@@ -678,7 +678,8 @@
 					5 ;"contest-other4",
 					5 ;"contest-other5",
 					5 ;"contest-other6",
-					5 ;"contest-other7"
+					5 ;"contest-other7",
+					5 ;"contest-other8"
```
Comment (FlameArrow57) [inline code/obj/decal/poster.dm:682]: This is for contest posters I think. poster_rand may be the place?

Comment (Fungal-Infection) [reply code/obj/decal/poster.dm:682]: good point, ill change that a tad bit later

**Comment by Fungal-Infection** [comment]
i made a mistake, and then immediately realized and had to fix it

**Comment by Fungal-Infection** [comment]
how does the QUICKFIX have a fail oh my goodness hold on

**Comment by Fungal-Infection** [comment]
it is now a bad icon operation aegh

**Comment by Fungal-Infection** [comment]
i will never touch postercode again 😭

**Comment by FlameArrow57** [comment]
wdym? Code seems fine, just adding to random_icon_states

**Comment by Fungal-Infection** [comment]
ive got nO idea

**Comment by FlameArrow57** [comment]
If it's because of the error that's showing up, that's something outside of this PR, not caused by it

**Pull request closed without merging**